jerry-debugger/src/lib/debugger-client.ts

```diff
@@ -0,0 +1,47 @@
+// Copyright JS Foundation and other contributors, http://js.foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import WebSocket from 'ws';
+
+export class JerryDebugger {
+  readonly host: string;
+  readonly port: number;
+  private _socket: WebSocket | undefined = undefined;
+  private _connectPromise: Promise<void>;
+
+  constructor(address: string) {
```

Comment:
I'd instead pass an "options object" here and add a `interface ...Options { host: string; port: number; ... }` like you have in the other file.

This is a bit more flexible compared to passing arguments, esp. with an eye on future extensibility.